### DISCUSSION TITLE
请问如何进行图片分页排版

### DISCUSSION THREAD

**Discussion by Heqilian** [tuna/thuthesis#756] [Q&A] [answered]
使用subcaption或者minipage等在一个总图中放置多个子图的时候，latex会自动保证图片连续性放在同一页，这样导致图文衔接出现断层。
请问如何让图片的前几张显示在上一页还有留白的位置，下一页继续显示图片后几张，而不是用一个全新页面来显示图片？

**Comment by zepinglee** [comment, marked as answer]
> 这样导致图文衔接出现断层。

学术论文一般不会在一个 `figure` 环境中分页。需要详细描述图的文字可以写在 caption 中。
